sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/StorageImplUtils.java

```diff
@@ -370,8 +347,12 @@ public static String convertStorageExceptionMessage(String message, HttpResponse
                 && response.getHeaders().getValue(ERROR_CODE) != null) {
                 // Use a constant compiled Pattern as the match pattern is always the same and String.replaceFirst
                 // will compile the match String into a Pattern internally on each call.
```

Comment:
nit: we could get rid of these comments (not necessary to complete this PR though)